### DISCUSSION TITLE
How to handle recipients conflict of interest?

### DISCUSSION THREAD

**Discussion by lbattistella1** [globaleaks/globaleaks-whistleblowing-software#3865] [Q&A]
### Proposal

Is it possibile, if not already present (but I did not find anything similar), to introduce a function that allows to eliminate one of the recepients (e.g. in case of conflict of interest)? Thank you 

### Motivation and context

It can be useful in case the issue of the concern is related to one of the recepients (e.g conflict of interest). Thank you

**Comment by evilaliv3** [comment]
Hello @lbattistella1 

Thank you for your question.

There are two ways the software support handling situations of conflict of interest:

1. You can let the whistleblower choose to which recipient address the report
2. You can define a question acting as a discriminant for the conflict of interest.

To implement option 1 you should enable the recipient selection con the reporting Channel
To implement option 2 you should define a question of type Select Box and setting that in the case of a specific answer to the question the recipient is changed.

For example you can have a question like:
"Do you consider a situation of conflict of interest exist within the whistleblowing office?" yes/no, and in case of yes you could enable the opzion to assing a new alternative recipient (e.g. the full supervisory body)

**Comment by lbattistella1** [comment]
Good afternoon Giovanni,

 

I think we will implement the first solution you suggested. 

Thank you for the useful answer

 

Best regards

 

 



Lorenzo Battistella

ESG Analyst

 

Fratelli Cosulich S.p.A.

Molo Ponte Morosini, 41
16126 Genova

Ph.: +390102546912

Email: ***@***.***

Site: https://www.cosulich.com/

 <https://www.linkedin.com/company/fratelli-cosulich/> 

I suoi dati personali saranno trattati in conformità al Regolamento Europeo n. 679/2016 (c.d. “GDPR”), in vigore a decorrere dal 25 maggio 2018. A questo proposito, la invitiamo a prendere visione dell'apposita ​informativa privacy <http://www.cosulich.com/privacy_policy.php> ​Le informazioni contenute in questo documento sono strettamente riservate e ad uso esclusivo del destinatario. Fratelli Cosulich S.p.A. ha adottato il proprio Modello di Organizzazione, Gestione e Controllo ai sensi del D.lgs. 8 giugno 2001, n. 231 ed il proprio Codice Etico. Fratelli Cosulich S.p.A. presta la propria attività esclusivamente nel rispetto di quanto previsto dal Modello e dal Codice Etico e richiede ai terzi l’osservanza di tali documenti, dei quali invita a prendere visione, in quanto oggetto di costante aggiornamento, sul sito www.cosulich.com/Mod_231.pdf <http://www.cosulich.com/Mod_231.pdf> 

Your personal data will be processed in compliance with the EU General Data Protection Regulation no. 679/2016 (“GDPR”), in force since May 25, 2018. ​On the topic, please see our privacy policy <http://www.cosulich.com/privacy_policy.php> . All information in this document is confidential and for exclusive use of the addressee only. Fratelli Cosulich S.p.A has adopted an Organizational, Management and Control Model pursuant the Italy Law Decree n. 231 dated June 8th, 2001 ("231 Model") and a Code of Ethics. Fratelli Cosulich S.p.A. operates according to provisions stated in the 231 Model and in Code of Ethics and request any concerned third party to comply with. These documents, regularly and properly updated are available here www.cosulich.com/Mod_231.pdf <http://www.cosulich.com/Mod_231.pdf> 

 

Da: Giovanni Pellerano ***@***.***> 
Inviato: lunedì 11 dicembre 2023 17:35
A: globaleaks/GlobaLeaks ***@***.***>
Cc: lbattistella1 ***@***.***>; Mention ***@***.***>
Oggetto: Re: [globaleaks/GlobaLeaks] How to handle recipients conflict of interest? (Discussion #3865)

 

Hello @lbattistella1 <https://github.com/lbattistella1> 

Thank you for your question.

There are two ways the software support handling situations of conflict of interest:

1.	You can let the whistleblower choose to which recipient address the report
2.	You can define a question acting as a discriminant for the conflict of interest.

To implement option 1 you should enable the recipient selection con the reporting Channel
To implement option 2 you should define a question of type Select Box and setting that in the case of a specific answer to the question the recipient is changed.

For example you can have a question like:
"Do you consider a situation of conflict of interest exist within the whistleblowing office?" yes/no, and in case of yes you could enable the opzion to assing a new alternative recipient (e.g. the full supervisory body)
